Assignment8/Assignment8/Assignment8/MainWindow.xaml.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Microsoft.Win32;
+using System;
+using System.IO;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Threading;
+
+namespace Assignment8
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private readonly DispatcherTimer dispatcherTimer = new();
+        
+        private DateTime _StartTime = DateTime.Now;
+        private string _Description = null!;
+        private bool _IsStopped = true;
+        private int entryNumber = 1;
+
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            dispatcherTimer.Tick += new EventHandler(DispatcherTimer_Tick!);
+            dispatcherTimer.Interval = TimeSpan.FromSeconds(1);
+        }
+
+        private void DispatcherTimer_Tick(object sender, EventArgs e)
+        {
+            Timer.Text = (DateTime.Now - _StartTime).ToString(@"hh\:mm\:ss");
+        }
+
+        private void StartAndStop_Click(object sender, RoutedEventArgs e)
+        {
+            if (StartAndStop.Content.ToString() == "Start new timer")
+            {
+                _StartTime = DateTime.Now;
+                dispatcherTimer.Start();
+                _IsStopped = false;
+                StartAndStop.Content = "Stop timer";
+                StartAndStop.Background = (SolidColorBrush)new BrushConverter().ConvertFrom("#ff0000");
```

Comment:
This works but there is a simpler solution.
```suggestion
                StartAndStop.Background = new SolidColorBrush(Color.FromRgb(0xff, 0x00, 0x00));
```